src/node.js

```diff
@@ -0,0 +1,21 @@
+'use strict'
+
+export default class Node{
+  constructor(data) {
+    this.data = data
+    this.next = null
+
+  }
+
+  setNext(data) {
+    this.next = data
+    return this
+  }
+
+  getNext() {
+    if (this.next) {
```

Comment:
You're the best @rachel-ftw... Thanks for your detailed feedback, the ninja training is strong in this one... lol